packaging/generate_package.sh

```diff
@@ -0,0 +1,50 @@
+#! /bin/bash
+
+VARIANT=$1
+if [[ ${VARIANT} == "deb" ]]; then
+    if [ -z "$3" ]; then
+        echo "Error, OS name and architecture missing for deb package type"
+        exit 1
+    else
+        OS=$2
+	ARCH=$3
+    fi
+fi
+
+VERSION_NO_SUFFIX="1.0.0"
+VERSION_SUFFIX="dev"
+VERSION="$VERSION_NO_SUFFIX"-"$VERSION_SUFFIX"
+
+# Using CMAKE_BINARY_DIR uses an absolute path and will break cross-vm building/download/make functionality
+BUILD_DIR="../"
+
+VENDOR="EOSNetworkFoundation"
+PROJECT_PREFIX="ANTELOPEIO"
+PROJECT="DUNE"
```

Comment:
We can't name our project `dune`  without having interference with an existing package.

See https://packages.ubuntu.com/kinetic/dune

Maybe rename as `dune-eos`? 

Also, package names should probably be lowercase, though it seems `apt` is case insensitive.

Reply:
Thanks, changed